Makefile

```diff
@@ -17,5 +17,5 @@ clean:
 
 .PHONY: release
 release:
-	go get github.com/mitchellh/gox
-	gox -ldflags="$(FLAGS)" -output="build/{{.Dir}}-{{.OS}}-{{.Arch}}" -osarch="linux/amd64 windows/amd64" .
+	go install github.com/mitchellh/gox
```

Comment:
I think you want `@latest` on the end here so it doesn't make any changes to go.mod etc. There's no need to change the deps for the binary itself since we're just using `gox` as a build tool here.

Reply:
Ah cool! Done in https://github.com/buildkite/ecs-run-task/pull/68/commits/2f30dc8828557647764c44d231313145373b01cf